### PR TITLE
CT-1439 refresh tables information test

### PR DESCRIPTION
**Pull request by vojtabiberle**
Jira: [CT-1439](https://keboola.atlassian.net/browse/CT-1439)
KBC: https://github.com/keboola/connection/pull/5104

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

[CT-1439]: https://keboola.atlassian.net/browse/CT-1439?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
ten ws mi prijde uplne zbytency

File: tests/Backend/ExternalBuckets/SnowflakeBucketsRefreshTablesInformationTest.php [hunk 1]
```diff
@@ -4,18 +4,87 @@
...
+        $createdTableRows = 10;
+
+        $workspaces = new Workspaces($this->_client);
+
+        $workspace = $workspaces->createWorkspace(
+            [
+                'backend' => self::BACKEND_SNOWFLAKE,
+            ],
+        );
+        $wsBackend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
```
[the remark this answers is not in the document]
Comment (jirkasemmler) [reply tests/Backend/ExternalBuckets/SnowflakeBucketsRefreshTablesInformationTest.php:39]: [tady](https://keboolaglobal.slack.com/archives/C0553EM2RGX/p1717487610163069?thread_ts=1717148722.586599&cid=C0553EM2RGX) jsme se dohodli, ze ten WS je ok. Jen tady ten radek je zbytecny ne? ta promenna `wsBackend` se nikde nepouziva

Comment (jirkasemmler) [inline tests/Backend/ExternalBuckets/SnowflakeBucketsRefreshTablesInformationTest.php:39]:
```suggestion
```

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
jen zrusit tu promennou

File: tests/Backend/ExternalBuckets/SnowflakeBucketsRefreshTablesInformationTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (vojtabiberle) [reply tests/Backend/ExternalBuckets/SnowflakeBucketsRefreshTablesInformationTest.php]: pravda, odstraněno

**Comment by ondrajodas** [comment]
v klientovi by nemělo být omezení na branch... když to teď zavolám přes BranchAwareClient tak to bude volat url `branch/ID/branch/ID/buckets...`

**Pull request merged**